sphinx-docs/usermanual/userman_admin.rst

```diff
@@ -0,0 +1,331 @@
+
+* If you have forgotten the username/password combination, simply delete this version of KA Lite (delete the “ka-lite” folder that you downloaded during the installation guide steps for your system and then redo the installation steps in the Install Guide). If it is critical that you are able to login with your credentials, :doc:`../contact` and we can help you manually reset your login information.
+
+Once you’ve logged in, the next step in the setup process is registering your device with the KA Lite Hub.
+
+Registering Your Device with the Hub
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By registering your device with FLE, you can sync data back with our central data hub. This is useful for many reasons:
+
+    #. a project administrator can manage user accounts and view usage data from afar, without needing to physically visit the offline device.
+    #. usage data syncing back can inform the FLE team of multiple users in a certain geographic region, and we can connect people that might be able to help one another
+    #. it helps FLE and our partners understand where and how the software is being used, so we can keep adding features that support you! 
+
+You will have two options: 
+
+    #. **One click registration**. This is the perfect option for individual users who just want to get KA Lite up and running fast, and don't need online access to data. This allows you to get the registration process over in one-click without worrying about creating a login that you're never going to use. **Caution**: if you choose to one-click register, you will unable to register with online access to data later. (If you chose this option by accident and would like to start over, please see **PUT LINK ON HOW TO START OVER**).
+
+
+    #. **Register with online access to data**. Choose this option if you're an administrator of larger projects. This option allows you to access your uploaded data and connect multiple installations to the same account. 
+
+How to register your device with online access to data
+########################################################
+
+
+**system currently down, this needs updating later**
+
+
+
+
+Post Registration Setup
+-------------------------------------------
+Now that you have registered successfully, it's time to configure your local KA Lite installation to suit your needs. If any terms like 'facility' or 'device' become confusing, feel free to reference the `Administrator Glossary`_ for a quick reminder.
+
+Create a Facility
+-------------------------
+KA Lite assumes that you are going to be using the software primarily in one place. This could be a school, a home, a community center, etc. We call this place a “facility”, and use it to help differentiate users who are syncing back data with our central data hub. In order to create a facility, follow the steps below.
+
+#. Log in to KA Lite.
```

Comment:
One problem that the nalanda students had was learning how to log in in the first place. Maybe we can add more screenshots here too?